Domain Connect Spec Draft.adoc

```diff
@@ -1583,29 +1586,34 @@ display to them the specific records that are being set in DNS.
 * If there are conflicts, either at the template or record level, the
 consent UX should warn the user about these conflicts. For templates,
 this would be services that would be disabled. For records, this would be
-records that would be overwritten. This could be progressively disclosed.
-
-Note: When re-applying the same template, ideally DNS Providers would not detect
-the conflict. Instead, the first template would be removed and the new
-instance applied. For most templates this is a benign operation, unless
-the template contains variables in host names. For consideration
-of this, see the section below.
+records that would be deleted or overwritten. This could be progressively disclosed.
 
 === Record Types and Conflicts
 
-The proposed handling of records and conflicts is as follows (if not
-otherwise specified, conflicts occur if the records have the same name):
+Conflict detection done by DNS provider prior to template application has to take
+into consideration specifics of each DNS record type. The rules outlined below shall
+assure predictable conflict resolution between DNS providers. Each rule applies to
+the records on the very same host, unless specifed otherwise.
+
+* CNAME records conflicts with all other records
+* any record type conflicts with existing CNAME record

```

Comment:
Not sure this line makes  sense

Reply:
I understand what you are saying; the wording is a bit offputting.

I would suggest: CNAME records conflict  with all other records, and any other records of any type conflict with an existing CNAME record.

Reply:
Updated as per suggestion in afcef45c002de75af61dba5cb6c71ae08092cd71